config/base.yml

```diff
@@ -738,12 +916,16 @@ Style/ColonMethodDefinition:
 Style/CommandLiteral:
   Enabled: true
   EnforcedStyle: mixed
```

Comment:
@ktopolski I just realized you changed this to `backticks`. But I changed it to `mixed` to match the latest version of master (not reflected on my fork).

Reply:
nice catch, thank you Hunter :)